docs/addons/addon-migration-guide.md

```diff
@@ -1,77 +1,54 @@
 ---
-title: Addon migration guide for Storybook 7.0
+title: Addon migration guide for Storybook 8.0
 ---
 
-Storybook 7 is our first major release in over two years. While Storybook’s addon API has not changed much in the past couple of years, addons require several changes for compatibility with Storybook 7. This guide will walk you through the upgrade process.
+We deeply appreciate the dedication and effort addon creators put into keeping the Storybook ecosystem vibrant and up-to-date. As Storybook evolves to version 8.0, bringing new features and improvements, this guide is here to assist you in migrating your addons from 7.x to 8.0. If you need to migrate your addon from an earlier version of Storybook, please first refer to the [Addon migration guide for Storybook 7.0](https://storybook.js.org/docs/addons/addon-migration-guide).
 
 <Callout variant="info">
 
 As we gather feedback from the community, we’ll update this page. We also have a general [Storybook migration guide](../migration-guide.md) if you’re looking for that.
 
 </Callout>
 
-## Dependencies
+## Updating Dependencies
```

Comment:
We use sentence case for headings, so this should be "Updating dependencies" and the other headings should be adjusted accordingly.